v5/okapi/tutorials/walkthrough/autonomous-movement-async.rst

```diff
@@ -9,76 +9,63 @@ Oftentimes the fastest way to move in autonomous involves actuating multiple sub
 (i.e. driving and raising/lowering a lift). This is made possible with 
 `Async Controllers <../../api/control/async/async-controller-factory.html>`_.
 
-To create an Async Controller for a given system, modify the below example to fit your subsystem
-(You'll most likely want more than one motor for your drivetrain).
+To create an Chassis Controller for a given system, modify the below example to fit your subsystem.
 
 .. highlight: cpp
 .. code-block:: cpp
    :linenos:
    :name: autonomous.cpp
 
-   const double drivekP = 1.0;
-   const double drivekI = 0.001;
-   const double drivekD = 0.1;
-   const int DRIVE_MOTOR_PORT = 1;
+   const int DRIVE_LEFT_MOTOR_PORT = 1;
+   const int DRIVE_RIGHT_MOTOR_PORT = 2;
   
-   okapi::Motor driveMotor (DRIVE_MOTOR_PORT);
-   auto driveController = okapi::AsyncControllerFactory::posPID(driveMotor, drivekP, drivekI, drivekD);
+   auto driveController = okapi::ChassisControllerFactory::create(DRIVE_LEFT_MOTOR_PORT, DRIVE_RIGHT_MOTOR_PORT);
   
-And then we'll duplicate this to add a lift subsystem:
+And then we'll add a lift subsystem as an Async Controller as opposed to a Chassis Controller:
```

Comment:
Why draw the comparison? Just say "And then we'll add a lift subsystem using an Async Controller"